src/AddOn.Episerver.Settings/Core/SettingsInitializationModule.cs

```diff
@@ -176,38 +175,38 @@ public void Uninitialize(InitializationEngine context)
     }
 
     /// <summary>
-    ///     Executed when the content is being deleted.
+    ///     Initializes the complete handler.
     /// </summary>
-    /// <param name="sender">The sender.</param>
-    /// <param name="e">The <see cref="DeleteContentEventArgs" /> instance containing the event data.</param>
-    private static void DeletingContent(object sender, DeleteContentEventArgs e)
+    /// <param name="sender"> The sender. </param>
+    /// <param name="e"> The <see cref="EventArgs" /> instance containing the event data. </param>
+    /// <exception cref="T:System.NotSupportedException">If the rootname is already registered with another contentRootId.</exception>
+    private static void InitCompleteHandler(object sender, EventArgs e)
```

Comment:
Move this down to under CreatingContent in order to not seem like an add/remove operation.

Reply:
Moved this last and included the code from the inlined handler so we only have one.